pages/api/v1/auth/actions/followUser.ts

```
@@ -0,0 +1,70 @@
+import messages from '~src/auth/utils/messages';
+import { firestore_db } from '~src/services/firebaseInit';
+import { IFollowEntry } from '~src/types';
+
+async function handler(req: NextApiRequest, res: NextApiResponse<MessageType>) {
+	storeApiKeyUsage(req);
+
+	if (req.method !== 'POST') return res.status(405).json({ message: 'Invalid request method, POST required.' });
+
+	// userId to follow
+	const { userId } = req.body;
+	if (isNaN(Number(userId)) || userId === null || userId === undefined) return res.status(400).json({ message: 'Missing or invalid user id in request body' });
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)